rollup.config.js

```diff
@@ -24,62 +21,26 @@ const config = {
     onwarn: console.warn
 }
 
-const locales = fs
-    .readdirSync(path.join(__dirname, 'src/i18n'))
-    .map(filename => path.parse(filename).name)
-    // Don't create a bundle for all locales in a development environment
-    .filter(locale => process.env.NODE_ENV === 'development' ? locale === 'en' : true)
-
-let bundles
-
-function createBundle(locale) {
-    const plugins = [
-        ...config.plugins,
-        alias({
-            entries: [
-                { find: '@reachfive/i18n', replacement: path.join(__dirname, `src/i18n/${locale}.js`) }
-            ]
-        })
-    ];
-
-    return [
-        {
-            ...config,
-            plugins,
-            output: { file: `build/main.${locale}.es.js`, format: 'es' },
-            external: dependencies
-        },
-        {
-            ...config,
-            plugins,
-            output: { file: `build/main.${locale}.cjs.js`, format: 'cjs' },
-            external: dependencies
-        },
-        // {
-        //     ...config,
-        //     plugins,
-        //     output: {
-        //         file: `build/main.${locale}.umd.js`,
-        //         format: 'umd',
-        //         name: 'reach5Widgets',
-        //         globals: {
-        //             '@reachfive/identity-core': 'reach5'
-        //         }
-        //     },
-        // }
-    ]
-}
-
-if (process.env.LOCALE) {
-    const locale = process.env.LOCALE
-
-    if (!locales.includes(locale)) {
-        throw new Error(`Invalid locale "${locale}" provided. The "LOCALE" environment variable must be one of: ${locales.join(', ')}.`)
-    }
-
-    bundles = createBundle(locale)
-} else {
-    bundles = locales.reduce((bundles, locale) => [...bundles, ...createBundle(locale)], [])
-}
-
-export default bundles
+export default [
+    {
+        ...config,
+        output: { file: 'build/main.es.js', format: 'es' },
+        external: dependencies
+    },
+    {
+        ...config,
+        output: { file: 'build/main.cjs.js', format: 'cjs' },
+        external: dependencies
+    },
+    // {
```

Comment:
Do we want to keep these comments ?

Reply:
Yes, it will be useful to fetch the SDK from a CDN. But since it's not working yet, I've commented the configuration (it will be fixed soon).